examples/limitrange/valid-pod.json

```diff
@@ -1,20 +1,19 @@
 {
   "id": "valid-pod",
   "kind": "Pod",
-  "apiVersion":"v1beta2",
```

Comment:
Should we actually be updating to v1beta3 if we're updating at all? cc @roberthbailey